api/albums-controller.js

```diff
@@ -133,5 +154,7 @@ module.exports = {
   getSharedAlbums,
   initializeCache,
   joinAlbum,
+  joinPendingAlbums,
   shareAlbum,
-}
+  getShareTokens,
```

Comment:
Doesn't look like we need to export this, yeah? Since it's coming from ./api/user.js